katalog/dual-nginx/README.md

```diff
@@ -1,28 +1,27 @@
-
 # Ingress Dual Nginx
 
-Ingress Nginx is an Ingress Controller for [NGINX](https://nginx.org) webserver and reverse proxy, it manages Nginx in a Kubernetes native manner. This package deploys 2 Nginx Controllers, one `external` to serve public traffic, one `internal` to serve internal traffic, for clusters created with kubeadm.
+Ingress Nginx is an Ingress Controller for [NGINX](https://nginx.org) webserver and reverse proxy, it manages Nginx in 
+a Kubernetes native manner. This package deploys 2 Nginx Controllers, one `external` to serve public traffic, 
+one `internal` to serve internal traffic, for clusters created with kubeadm.
```

Comment:
Does the cluster needs to be strictly created using `kubeadm` to use this module?

Reply:
I don't think so